### PR TITLE
Issue/17

### LINKED ISSUE #17: !!
Update devs to sudos

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
- Allow upgrades to sudos

### CODE REVIEW

File: contracts/Mesh.sol [hunk 1]
```diff
@@ -40,6 +48,7 @@ contract Mesh is ERC721, Ownable {
     }
 
     function setProbation(uint256 index, uint256 value) external onlyOwner() {
+        require(value > 0);
         mesh[index].probationSeconds = value;
```
Comment (bazmatic) [inline contracts/Mesh.sol:53]: Should we not also check that the record exists?
```suggestion
        require(index <= totalSupply, "No such Meshie");
        mesh[index].probationSeconds = value;
```

File: contracts/ERC721.sol [hunk 2]
```diff
@@ -141,9 +141,7 @@ contract ERC721 is Context, ERC165, IERC721, IERC721Metadata {
         uint256 tokenId
     ) public virtual override {
         //solhint-disable-next-line max-line-length
-        require(_isApprovedOrOwner(_msgSender(), tokenId), "ERC721: transfer caller is not owner nor approved");
-
-        _transfer(from, to, tokenId);
+        revert("Not so fast");
```
Comment (RalphLavelle) [inline contracts/ERC721.sol:144]: Not clear to me what the change means here

File: contracts/ERC721.sol [hunk 3]
```diff
@@ -166,36 +164,7 @@ contract ERC721 is Context, ERC165, IERC721, IERC721Metadata {
...
-     */
-    function _safeTransfer(
-        address from,
-        address to,
-        uint256 tokenId,
-        bytes memory _data
-    ) internal virtual {
-        _transfer(from, to, tokenId);
-        require(_checkOnERC721Received(from, to, tokenId, _data), "ERC721: transfer to non ERC721Receiver implementer");
+        revert("Not so fast");
```
Comment (RalphLavelle) [inline contracts/ERC721.sol:167]: Same here

File: contracts/Mesh.sol [hunk 4]
```diff
@@ -49,6 +49,7 @@ contract Mesh is ERC721, Ownable {
 
     function setProbation(uint256 index, uint256 value) external onlyOwner() {
         require(value > 0);
+        require(index <= totalSupply, "No such Meshie");
```
Comment (rellfy) [inline contracts/Mesh.sol:52]: Doesn't seem to be inclusive (we mint starting at zero)
```suggestion
        require(index < totalSupply, "No such Meshie");
```

**Pull request closed without merging**